python/hail/methods/misc.py

```diff
@@ -307,3 +307,40 @@ def wrap_input(interval):
     intervals = [wrap_input(x)._jrep for x in intervals.value]
     jmt = Env.hail().methods.FilterIntervals.apply(ds._jvds, intervals, keep)
     return MatrixTable(jmt)
+
+@typecheck(mt=MatrixTable, base_pairs=int)
+def window_by_locus(mt: MatrixTable, base_pairs: int) -> MatrixTable:
+    """Collect arrays of row and entry values from previous variants.
+
+    .. include:: ../_templates/req_tvariant.rst
```

Comment:
req_tvariant covers this

Reply:
I know it covers it, my point was that req_tvariant is overly restrictive since `alleles` aren't touched. But I'm fine with not explaining that while it's in experimental phase.